integrationTests/chainSimulator/bridge/common.go

```diff
@@ -170,7 +170,8 @@ func deposit(
 
 	depositArgs = depositArgs +
 		"@" + hex.EncodeToString([]byte("deposit")) +
-		"@" + hex.EncodeToString(receiver)
+		"@" + hex.EncodeToString(receiver) +
+		"@2710@68656c6c6f@0000000131"
```

Comment:
Please split this line in multiple "@" + .... and add a comment for each arg

Reply:
Same, didn't mean to push this. Rolled back